code/game/objects/items/weapons/twohanded.dm

```diff
@@ -256,6 +257,8 @@
 		return 1
 
 /obj/item/weapon/twohanded/dualsaber/wield(mob/living/carbon/M) //Specific wield () hulk checks due to reflection chance for balance issues and switches hitsounds.
+	sharpness = IS_SHARP
+	w_class = w_class_on
```

Comment:
put these AFTER the hulk dna check, otherwise the item would have the wielded properties while being unwielded.